src/edu/stanford/nlp/sempre/InteractiveServer.java

```diff
@@ -0,0 +1,334 @@
+package edu.stanford.nlp.sempre;
```

Comment:
yeah ideally. but then Deriviation.score, and some Session fields need to be public.